dts/arm/nxp/nxp_mcxc_common.dtsi

```diff
@@ -67,7 +67,7 @@
 			status = "disabled";
 			#address-cells = <1>;
 			#size-cells = <1>;
-			fsec = <0xff>;
+			fsec = <0xfe>;
```

Comment:
Is this same issue seen with JLink?

Reply:
Hi @mmahadevan108 and @michal-smola ,
JLink manages the flash security settings in the FSEC field to help prevent developers from accidentally locking the flash, and bricking the device.  The default flash loader will ignore the FSEC setting in the application image, and will program FSEC to leave the device unsecured.  If a user wants the FSEC locked, then a parameter must be passed to the JLink algorithm to accept the FSEC setting in the image.  Which is why this is not typically seen when using the JLink runner.

Reply:
@michal-smola @DerekSnell, similar to JLink, is there a way to manage this in the Linkserver runner rather than make a system wide change

Reply:
Having the device unsecured (FSEC = 0xfe) by default is inline with mcux sdk. It was rather a bug to have it secured. Change Linkserver behavior such that it is able to unsecure the mcu would be nice, but probably better to do it in another PR.